### ISSUE TITLE
Teksten

### ISSUE THREAD

**Issue by evanzummeren** [vpro/tinderlicht#2]

**Comment by kingjaynl** [comment]
**Scherm 1**

Met VPRO Tinderlicht kun je op zoek naar een date voor de [Tegenlicht Meet Up ](https://dezwijger.nl/programmareeks/tegenlicht-meet-up/) over de aflevering [Tinder love](http://tegenlicht.vpro.nl/afleveringen/2015-2016/tinder-love.html). Om je te kunnen matchen, moet je inloggen met Twitter of Facebook.

**Comment by kingjaynl** [comment]
Scherm 2 (Facebook)

Hoi [VOORNAAM],

Om fans van het programma aan elkaar te matchen, moet je wel fan van het programma zijn natuurlijk. Als je VPRO Tegenlicht nog niet volgt op Facebook, dan kun je dat nu alsnog doen.

[LIKE BUTTON]

Scherm 2 (Twitter)

Hoi [TWITTER_HANDLE],

Om fans van het programma aan elkaar te matchen, moet je wel fan van het programma zijn natuurlijk. Als je @vprotegenlicht nog niet volgt op Twitter, dan kun je dat nu alsnog doen.

[VOLG BUTTON]

**Comment by kingjaynl** [comment]
# Scherm 3

**Leeftijd**
Om je zo goed mogelijk te kunnen matchen, willen we graag een paar gegevens van je weten. Die gebruik we alleen om je te kunnen matchen en we gebruik deze gegevens nergens anders voor.

Mogen we weten hoe oud je bent?

[INVULVELD]

**Comment by kingjaynl** [comment]
#  Scherm 4

**Wat zoek je?**
Het helpt ook om te weten wat je geslacht is en waar je naar op zoek bent.
Wat is je geslacht? [man/vrouw/transgender]
Wie zoek je? [man/vrouw/maakt niet uit]

**Comment by kingjaynl** [comment]
# Scherm 5

**Wat is je mailadres?**
Om je op de hoogte te brengen van een match, willen we je graag mailen. 

_We gebruiken je mailadres voor geen andere doel dan voor het versturen van notificaties over nieuwe matches en delen je mailadres niet met derden._

Wat is je mailadres?
[INVULVELD]

**Comment by evanzummeren** [comment]
Paar kleine edits

**Comment by evanzummeren** [comment]
Btw hoe match je eigenlijk transgenders in combinatie met je voorkeur? En waarom zou je dit eigenlijk als apart gender opnemen, ik neem aan dat een transgender zich met een man of een vrouw identificeert, en niet met allebei (hoewel dat biologisch misschien wel het geval is).

**Comment by kingjaynl** [comment]
# Scherm 6

**Omschrijf jezelf**
Wat verwacht je van een afspraakje tijdens een Tegenlicht Meet Up en wie ben je? Kijk je elke week VPRO Tegenlicht of ben je meer een zondagskijker?

[Gebruik maximaal 200 tekens om jezelf te omschrijven voor potentiële matches.]

**Comment by kingjaynl** [comment]
# Scherm 7

**Tegenlicht Meet Up**
Naar welke Tegenlicht Meet Up ga je?

[Landelijk in Pakhuis de Zwijger, Amsterdam (17 februari) ]

Ik ga naar een lokale Tegenlicht Meet Up, namelijk in:  _(pulldownmenu)_
[Amersfoort (Stasblad033, 17 februari)]
[Vlaardingen (KADE40, 17 februari)]
etc. etc.

**Comment by kingjaynl** [comment]
En transgender als optie omdat wij enorm 2016 zijn en een programma over future affairs en er steeds meer transgenders zijn. Enzo.

**Comment by kingjaynl** [comment]
# Geen matches

**Kom snel terug**
Je hebt iedereen in ons bestand beoordeeld! Wie weet melden zich snel weer nieuwe mensen aan, dus kom binnenkort nog eens terug. Wie weet vind je dan je ideale match! Je kunt natuurlijk ook gewoon naar een Tegenlicht Meet Up gaan en wie weet spreek je iemand na afloop tijdens de borrel. Proost!

**Comment by evanzummeren** [comment]
Ja, maar zelfs de technische uitwerking van transgenders? Ik weet niet hoe ik dit in het matching moet verwerken? Transgenders aan iedereen tonen?

**Comment by evanzummeren** [comment]
Jullie zijn een match! Je kan de ander bevrienden via Facebook. Je kan dan via de chatfunctie van Facebook met elkaar in gesprek komen. Als jullie besluiten om samen naar de Tegenlicht Meet Up te gaan, dan kan je ons dat via deze speciale aanmeldpagina laten weten. We reserveren dan een tafeltje voor je en jullie krijgen allebei een gratis glaasje wijn!

**Comment by kingjaynl** [comment]
Jullie zijn een match! Als je vrienden met elkaar wordt via Facebook, dan kun je daar verder chatten. Besluiten jullie om samen naar de landelijke Tegenlicht Meet Up in Pakhuis de Zwijger te gaan, dan kun je dat ons via de mail laten weten. Voor de eerste 10 koppels is een romantisch tafeltje met 2 glaasjes rode wijn gereserveerd.

Jullie zijn een match! Als je elkaar gaat volgen op Twitter, dan kun je daar verder chatten via direct messages. Besluiten jullie om samen naar de landelijke Tegenlicht Meet Up in Pakhuis de Zwijger te gaan, dan kun je dat ons via de mail laten weten. Voor de eerste 10 koppels is een romantisch tafeltje met 2 glaasjes rode wijn gereserveerd.

**Comment by kingjaynl** [comment]
Zinnetje bij de Matches:
"Ik ga samen met een match naar de landelijke Tegenlicht Meet Up"

aan: charley@dezwijger.nl
onderwerp: Ik kom samen met een match naar de Tegenlicht Meet Up
body: _(Als je samen met een match aan romantisch tafeltje wil genieten van een gratis glas rode wijn dan kun je je via deze mail daarvoor aanmelden. Voor de eerste 10 koppels wordt een tafel gereserveerd. Vermeld in deze mail even je eigen naam en de naam van je match. Je ontvangt dan een mail met meer informatie.)_

**Comment by evanzummeren** [comment]
Ik bedenk mij nu dat het misschien wel verstandiger is om hier een form van te maken. Dit soort klikbare email links werken wel wanneer je een 'echte' email client hebt, maar wanneer je van een webbased variant hebt dan werkt het wat minder lekker.

Ik heb daar vrijdagmiddag oid nog wel tijd voor, is in principe zo gepiept.

**Comment by kingjaynl** [comment]
okido

**Comment by kingjaynl** [comment]
volgens mij heb je nu alle teksten van me?

**Comment by evanzummeren** [comment]
Ja, eventueel nog het matchmailtje. (Als dat beveiligingstechnisch allemaal mogelijk is)

**Comment by kingjaynl** [comment]
onderwerp: Match via VPRO Tinderlicht!

Beste [VOORNAAM/TWITTER_HANDLE],

Gefeliciteerd! Je hebt een match via VPRO Tinderlicht. Ga snel naar de app om te kijken wie samen met jou naar de Tegenlicht Meet Up wil. Jullie kunnen met elkaar chatten via Facebook of Twitter en dan besluiten of je samen op stap gaat. Om te ontdekken wie je match is, surf je naar:

tegenlicht.vpro.nl/tinderlicht

Als jullie besluiten om samen naar de landelijke Tegenlicht Meet Up in Pakhuis de Zwijger (Amsterdam) te gaan, laat ons dat dan even weten. Voor de eerste 10 koppels hebben we een romantisch tafeltje klaar staan met twee gratis glaasje rode wijn.

Groet!
VPRO Tinderlicht
